### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by daviddrysdale**
# Checklist

- [x] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [x] I have written tests that cover the code changes.
  - [x] I have checked that these tests are run by
        [Cloudbuild](/cloudbuild.yaml)
  - [ ] I have updated [documentation](/docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

**Comment by daviddrysdale** [comment]
Individual commit messages give a bit more background, btw.

**Comment by daviddrysdale** [comment]
(Also to confirm: the test does indeed [catch the deadlock](https://github.com/project-oak/oak/runs/1754432030?check_suite_focus=true))

File: oak_runtime/src/lib.rs [hunk 1]
```diff
@@ -463,7 +463,7 @@ impl Runtime {
             if node_info.abi_handles.get(&candidate).is_none() {
                 debug!(
                     "{:?}: new ABI handle {} maps to {:?}",
-                    self.get_node_debug_id(node_id),
+                    node_info.get_debug_id(node_id),
```
Comment (juliettepretot) [inline oak_runtime/src/lib.rs:466]: Good catch, but also oh noes. 😬
I haven't worked with this multi threaded code in a while, but deadlocking the threads aren't really something I've started making peace with yet. It just seems so tricky to catch, it I don't like it

**Review by juliettepretot — APPROVED** [review]
LGTM :) 

+1 For the neat commit messages with informative descriptions, it's very nice

**Pull request merged**